charts/apisix/Chart.yaml

```diff
@@ -31,7 +31,7 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 1.4.0
+version: 1.5.0
```

Comment:
Since it will be a major breaking change. I recommend using v2.x for maintenance